### DISCUSSION TITLE
[Question not issue] Is it tested in Windows 11 ?

### DISCUSSION THREAD

**Discussion by daniellumertz** [bp2008/BetterClearTypeTuner#16] [Q&A] [answered]
Hey, not a issue just wanted to know if anyone have used it in win 11 and see if it works

**Comment by bushjt987** [comment, marked as answer]
Hey it works for me on 11.

**Comment by Hunanbean-Collective** [comment]
Just wanted to mention that it is still working on 23H2 Win11 Jan 2024, and that it is appreciated. Thank you
